### PR TITLE
clean up expired cache

### PR DESCRIPTION
**Pull request by StepanovDaniilOlegovich**
## Description
Open caches by filtered out cache names, iterate through its entries, find entries with date header older then seven days, delete them. Resolve done to riseplayerconfiguration

## Motivation and Context
This is for deleting old, unnecessary cache to increase cache storage and to keep it relevant

## How Has This Been Tested?
in apps with mapping to local file

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: src/rise-purge-cached-files.js [hunk 1]
```diff
@@ -31,14 +36,44 @@ RisePlayerConfiguration.PurgeCacheFiles = (() => {
...
+
+        if ( _compareDates( currentTimestamp, lastRequested )) {
+          cache.delete( entry.url );
+        }
+      })
+    })
+  }
+
+  function _compareDates( currentTimestamp, lastRequested ) {
+    const difference = currentTimestamp.valueOf() - new Date( lastRequested ).valueOf();
```
Comment (stulees) [inline src/rise-purge-cached-files.js:58]: @StepanovDaniilOlegovich The use of `valueOf` will work, but I think it's typically more common to use `getTime()` as [documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/valueOf) states:

> This method is functionally equivalent to the Date.prototype.getTime() method. This method is usually called internally by JavaScript and not explicitly in code.

File: src/rise-purge-cached-files.js [hunk 2]
```diff
@@ -1,27 +1,32 @@
-
 /* eslint-disable no-console, one-var, vars-on-top */
 
 RisePlayerConfiguration.PurgeCacheFiles = (() => {
   const componentCacheKeys = [ "rise-image", "rise-video" ];
+  const expiryTime = 1000 * 60 * 6 * 24 * 7;
```
Comment (stulees) [inline src/rise-purge-cached-files.js:5]: @StepanovDaniilOlegovich Is `6` a typo?  I think it should be `60` as in 60 mins in an hour.

Comment (StepanovDaniilOlegovich) [reply src/rise-purge-cached-files.js:5]: Yes thanks for pointing out

**Comment by StepanovDaniilOlegovich** [comment]
@stulees added tests, added requested changes!

**Pull request merged**